dotcom-rendering/src/types/config.ts

```diff
@@ -0,0 +1,69 @@
+export interface CommercialConfigType {
+	isPaidContent?: boolean;
+	pageId: string;
+	webPublicationDate?: number;
+	headline?: string;
+	author?: string;
+	keywords?: string;
+	section?: string;
+	edition?: string;
+	series?: string;
+	toneIds?: string;
+	contentType: string;
+	ampIframeUrl: string;
+}
+
+/** This type is not support by JSON-schema, it evaluates as `object` */
+export type ServerSideTests = {
+	[k: string]: 'variant' | 'control';
+};
+
+export type ServerSideTestNames = `${string}Control` | `${string}Variant`;
```

Comment:
Slight change from the previous implementation, which leads to a more accurate schema.